techniques/html/H85.html

```diff
@@ -98,16 +94,16 @@ <h2 class="box-h box-h-icon">About this Technique</h2>
-      <p>The <code class="language-html">optgroup</code> element should be directly inside the <code class="language-html">select</code> element, and the <code class="language-html">option</code>  elements directly inside the <code class="language-html">optgroup</code>. It is possible for a <code class="language-html">select</code> element to contain both single <code class="language-html">option</code> elements and <code class="language-html">optgroup</code> groups, though authors should consider if this is in fact the desired intent when using this. It is not possible to nest the <code class="language-html">optgroup</code> element, so only one level of grouping can be done within a <code class="language-html">select</code>.</p>
-      <p>If grouping information is essential to understanding the list, authors may define <code class="language-html">option</code> labels that can be understood even when the screen reader does not present the grouping information provided by <code class="language-html">optgroup</code>.</p>
+    <h2>解説</h2>
+    <p>このテクニックの目的は、選択リスト内の項目をグループ化することである。選択リストとは、複数を選択するリストやコンボボックスなどのフォームコントロールで使用できる値の集合である。多くの場合、選択リストには関連する選択肢のグループが含まれる。これらのグループは、「ダミー」のリスト項目で区切るのではなく、セマンティックに特定すべきである。例えば、水平線を作成するためのダッシュだけを含む <code class="language-html">option</code> 要素が挙げられる。選択肢のグループをセマンティックに特定することで、長いリストを視覚的に分割し、利用者が興味のあるものを見つけやすくする。</p>
+    <p>HTMLでは、<code class="language-html">select</code> 要素は複数を選択するリストとコンボボックスの両方を生成するために利用される。許可される様々な選択肢はそれぞれ <code class="language-html">option</code> 要素で示される。選択肢をグループ化するには、 <code class="language-html">optgroup</code> 要素を使用し、その要素内に関連する <code class="language-html">option</code> 要素を配置する。<code class="language-html">label</code> 属性でグループにラベルを付けることで、利用者がそのグループ内に何が含まれているのかを予想できるだろう。</p>
+      <p><code class="language-html">optgroup</code> 要素は <code class="language-html">select</code> 要素の直下に、<code class="language-html">option</code> 要素は <code class="language-html">optgroup</code> 要素の直下に配置すべきである。<code class="language-html">select</code> 要素には、単一の <code class="language-html">option</code> 要素と <code class="language-html">optgroup</code> 要素の両方を含めることができるが、コンテンツ制作者はこれを使用する場合、実際に意図されたものであるかどうかを検討すべきである。<code class="language-html">optgroup</code> 要素は入れ子にできないため、<code class="language-html">select</code> 要素内では 1 階層のみのグループ化が可能である。</p>
+      <p>グループ化情報がリストの理解に必要不可欠である場合、コンテンツ制作者はスクリーンリーダーが <code class="language-html">optgroup</code> によって提供されるグループ化情報を提示しない場合でも理解可能な <code class="language-html">option</code> のラベルを定義することができる。</p>
```

Comment:
読んでいて少しわかりにくい印象を受けたので、語順を並べ替えてみました。いかがでしょうか？

```suggestion
      <p>グループ化情報がリストの理解に必要不可欠である場合、スクリーンリーダーが <code class="language-html">optgroup</code> によって提供されるグループ化情報を提示しない場合でも、コンテンツ制作者は理解可能な <code class="language-html">option</code> のラベルを定義することができる。</p>
```